src/chi-vue/src/components/data-table-tooltip/DataTableTooltip.tsx

```diff
@@ -0,0 +1,29 @@
+import { Component, Vue, Prop } from 'vue-property-decorator';
+import Tooltip from '@/components/tooltip/tooltip';
+import { UTILITY_CLASSES } from '@/constants/classes';
+import { TOOLTIP_ANIMATION_DELAY } from '@/constants/constants';
+
+@Component({})
+export default class DataTableTooltip extends Vue {
+  @Prop() msg!: string;
+
+  tooltip = false;
+
+  render() {
+    return (
+      <div
+        onMouseenter={() => (this.tooltip = true)}
+        onFocus={() => (this.tooltip = true)}
+        onMouseleave={() => window.setTimeout(() => (this.tooltip = false), TOOLTIP_ANIMATION_DELAY)}
+        oBlur={() => window.setTimeout(() => (this.tooltip = false), TOOLTIP_ANIMATION_DELAY)}>
```

Comment:
```suggestion
        onBlur={() => RESPECTIVE_METHOD>
```
apart from the typo, could you create a method for the blur functionality? To avoid inline visibility of the function

![Screenshot 2021-07-12 at 12 13 16](https://user-images.githubusercontent.com/25136131/125271209-c937fd80-e30a-11eb-9d7f-763961cb5c00.png)

Reply:
is the approach deffinitive? I still see leftover tooltip elements in DOM.

Reply:
Tooltip emits `chiTooltipHidden` that gets triggered once the element is fully hidden. Could you use it instead of `setTimeout` ?

Reply:
Hey @janibolkvadze the issue of leftover is not related to timeout but it is an issue related to mouseEnter, mouseleave event. seems it is a native issue of VUEJS unfortunately but at least we are sure that we won't have many duplicated leftover which effects the performance.